routes/auth.js

```diff
@@ -0,0 +1,43 @@
+const express = require('express')
+const router = express.Router()
+const {
+  sgidService,
+  scopes,
+  randomnonce,
+} = require('../lib/sgid-client.service')
+const { formatData } = require('../lib/utils')
+
+router.get('/auth/login/:env?', (req, res) => {
+  const env = req.params.env || 'prod'
+  const authUrl = sgidService[env].authorizationUrl(
+    env,
+    scopes,
+    randomnonce
+  ).url
+  res.redirect(authUrl)
+})
+
+router.get('/callback', async (req, res) => {
```

Comment:
nit: this route isn't really an auth endpoint - would it make sense to put it in a separate routes file?